src/htrun/README.md

```diff
@@ -1,14 +1,14 @@
 # Development moved
 
-The development of Htrun has been moved into the [mbed-os-tools](../../src/mbed_os_tools) package. You can continue to use this module for legacy reasons, however all further development should be continued in the new package.
+The development of Htrun (mbed-host-tests) has been moved into the [mbed-greentea](https://github.com/ARMmbed/greentea/tree/master) repository under `src/htrun`. You can continue to use this module for legacy reasons, however all further development should be continued in the mbed-greentea repository.
```

Comment:
I think this paragraph to give some `warning` to the user as this source moved from a different repo and we will keep this notice until we remove the `htrun` source from `mbed-os-tools`

Reply:
Normally a warning is in an old place asking users to look at the new place, rather than in the new place itself like here. I don't mind too much for now as it's a minor detail.